Userland/Libraries/LibGL/GLContext.h

```diff
@@ -91,6 +91,11 @@ class GLContext final {
     NonnullRefPtr<Gfx::Bitmap> frontbuffer() const { return m_frontbuffer; };
     void present();
 
+    // Used by WebGL to preserve the clear values when implicitly clearing the front buffer.
+    FloatVector4 current_clear_color() const { return m_clear_color; }
```

Comment:
Might be better to add a `ContextParameter` for these values and get them via `gl_get_floatv()` instead of adding a new public API. But for now, can you just add a FIXME?